Makefile

```diff
@@ -107,14 +107,9 @@ else
 MAKE_J:=-j$(shell nproc)
 endif
 
-CMAKE_DIR=$(ROOT)/src
-
-CMAKE_FILES= \
-	src/CMakeLists.txt \
-	src/VecSim/spaces/CMakeLists.txt \
-	cmake/common.cmake \
-	cmake/gtest.cmake \
-	cmake/clang-sanitizers.cmake
+CMAKE_DIR=$(ROOT)
```

Comment:
CMAKE_DIR=$(ROOT)
CMAKE_TEST_DIR=$(ROOT)/tests
CMAKE_UNIT_TESTS=$(CMAKE_TEST_DIR)/unit
CMAKE_BENCHMARK_DIR=$(CMAKE_TEST_DIR)/benchmark

Reply:
removed, not in use